### PR TITLE
Add umongo

### PR DESCRIPTION
**Pull request by whophil**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by whophil** [comment]
@lafrech I listed you as a maintainer here, please confirm if you are OK with this!

**Comment by lafrech** [comment]
Approved

**Comment by whophil** [comment]
@conda-forge-admin, please ping team

File: recipes/umongo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{% set name = "umongo" %}
+{% set version = "3.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://github.com/Scille/umongo/archive/refs/tags/{{ version }}.tar.gz"
```
Comment (ocefpaf) [inline recipes/umongo/meta.yaml:9]: I'm pretty sure the auto updating bot will break with these `"`. We also don't need them, can you remove them?

**Pull request merged**